src/vs/workbench/parts/files/electron-browser/fileCommands.ts

```diff
@@ -353,6 +353,7 @@ CommandsRegistry.registerCommand({
 });
 
 function revealResourcesInOS(resources: URI[], windowsService: IWindowsService, notificationService: INotificationService, workspaceContextService: IWorkspaceContextService): void {
+	// 54483: Check with @Isi
```

Comment:
@sandy081 this is fine. That aciton should only be enabled for file schemas, because other shcemas can not be reveal in the underlying OS.
I will check the enablement of that action and might push some commints on master - just adding preconditions.

Reply:
Great. Thanks 👍